mtp_cashbook/templates/core/index.html

```diff
@@ -1,14 +1,22 @@
 {% extends "base.html" %}
 
-{% block title %}Yo{% endblock title %}
-
 {% block content %}
-<div class="row">
-  <div class="small-12 columns moj-wrapper">
-    <h1>
-      You are logged in!
-    </h1>
-    <p><a href="{% url 'auth:logout' %}">Log out</a></p>
-  </div>
-</div>
+  <h1>Money sent to prisoners</h1>
+
+  <ul class="SummaryBoxes">
+    <li class="SummaryBoxes--unprocessed">
+      <a href="#TODO">
+        <h2>New</h2>
+        <p><strong>54</strong> Payments to credit</p>
+      </a>
+    </li>
+
+    <li>
+      <a href="#TODO">
+        <h2>History</h2>
+        <p><span>Payments</span> credited by you</p>
+      </a>
```

Comment:
Be nice to translate these, even though we don't think we need other languages yet, because it'll make @leighmoney's life easier later.

Rough examples of how I'd do these blocks, as discussed (though no guarantees that this exact code will work):

```
{% blocktrans with payments_to_credit="<strong>{{ payments }}</strong>"  %}
{{ payments_to_credit }} payments
{% endblocktrans %}

{% blocktrans with highlight="<span>" endhighlight="</span>" %}
{{ highlight }}Payments{{ endhighlight }} credited by you
{% endblocktrans %}
```